app-modules/engagement/resources/views/components/filters.blade.php

```diff
@@ -100,4 +100,11 @@ class="!focus:ring-trout-700 text-advising-app-black-950 dark:text-advising-app-
             Open Service Requests
         </span>
     </label>
+
+    <label class="flex items-center">
+        <x-filament::input.checkbox wire:model.live="filterMemberOfCareTeam" />
+        <span class="ml-2">
+            Member of Care team
+        </span>
+    </label>
```

Comment:
Based on the screenshot, I believe Product would like this above My Subscriptions. Let's move it there.